scipy/stats/_morestats.py

```diff
@@ -1135,7 +1135,19 @@ def _boxcox_inv_lmbda(x, y):
     return np.real(-num / np.log(x) - 1 / y)
 
 
-def boxcox_normmax(x, brack=None, method='pearsonr', optimizer=None):
+class BigFloat:
+    def __repr__(self):
+        return "BIG_FLOAT"
```

Comment:
I do think this is more clear in the rendered documentation than `ymax=None` would have been.

<img width="789" alt="image" src="https://github.com/scipy/scipy/assets/6570539/33301dc5-bb1f-4b41-98a7-de17b8f68f5c">

If you can think of an even more descriptive string than "BIG_FLOAT", we can consider it.

Reply:
I think "BIG_FLOAT" is very clear, let's use it.